page/page.njk

```diff
@@ -110,19 +99,22 @@
 {% endblock %}
 
 {% block bodyEnd %}
-    {% include "modal-timeout.njk" %}
-    {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
-    <script nonce="{{ cspNonce }}">
-        window.CICA = {
-            SERVICE_URL: '{{ CW_URL }}',
-            ANALYTICS_TRACKING_ID: '{{ CW_GA_TRACKING_ID }}'
-        };
-    </script>
-    <script nonce="{{ cspNonce }}" src="/govuk-frontend/all.js"></script>
-    <script nonce="{{ cspNonce }}">window.GOVUKFrontend.initAll()</script>
-    <script nonce="{{ cspNonce }}" src="https://code.jquery.com/jquery-3.6.3.min.js" integrity="sha256-pvPw+upLPUjgMXY0G+8O0xUf+/Im1MZjXxxgOcBQBXU=" crossorigin="anonymous"></script>
-    <script nonce="{{ cspNonce }}" src="/dist/js/autocomplete.min.js"></script>
-    <script nonce="{{ cspNonce }}" src="/dist/js/bundle.js?{{CW_APP_VERSION}}"></script>
+  {% include "modal-timeout.njk" %}
+  {# Run JavaScript at end of the <body>, to avoid blocking the initial render. #}
+  <script nonce="{{ cspNonce }}">
+    window.CICA = {
+      SERVICE_URL: '{{ CW_URL }}',
+      ANALYTICS_TRACKING_ID: '{{ CW_GA_TRACKING_ID }}'
+    };
+  </script>
+  <script type="module" nonce="{{ cspNonce }}" src="/govuk-frontend/all.js?{{CW_APP_VERSION}}"></script>
+  <script type="module" nonce="{{ cspNonce }}">
+    import {initAll} from '/govuk-frontend/all.js?{{CW_APP_VERSION}}'
+    initAll()
```

Comment:
missing semi-colon!!!

Reply:
just following the instructions though. https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/758bfc7bd8c33fbe4f41994e60891903b84f76cd/page/blank.njk#L7